backend/prisma/migrations/20240702184800_posts_image_tables/migration.sql

```diff
@@ -0,0 +1,26 @@
+-- CreateTable
+CREATE TABLE "Post" (
+    "id" SERIAL NOT NULL,
+    "title" TEXT NOT NULL,
+    "content" TEXT NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
```

Comment:
when a post is created, is it also the same time when it's updated? If not, then shouldn't this be nullable?

Reply:
Correct, later on i wanted to implement for a user to be able to update and edit posts.

Reply:
updatedAt is now defined as DateTime?, indicating that it is optional.
These changes ensure that the updatedAt field is nullable at creation but will be updated with the current timestamp whenever the post is updated.